pxtlib/spriteutils.ts

```diff
@@ -647,6 +647,51 @@ namespace pxt.sprite {
         return result;
     }
 
+    export function addMissingTilemapTilesAndReferences(project: TilemapProject, asset: ProjectTilemap) {
```

Comment:
i think there's one more instance of both of these functions in the asset editor's `editor.tsx`